functions-and-scope.js

```diff
@@ -16,6 +16,15 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 
 // ---- Verwachte uitkomst: 6
 
+totalCumloude = 0
+i = 0
+
+for (i in grades) {
+    if (i => 8)
+    totalCumloude + 1 
```

Comment:
Dit doet op dit moment niets. Wellicht had je dit gewild
```suggestion
    totalCumloude++; 
```